### ISSUE TITLE
Changing audio track

### ISSUE THREAD

**Issue by ghost** [oaprograms/lingo-player#5]
Thank you for this extension: this is exactly what I was looking for. The translation works really well. 
However, I cannot choose an audio track. There are some movies with different audio tracks, and there is no way to change them...

**Comment by dezdem** [comment]
you have to use mkvtoolnix to repack video w/o unnecessary tracks. also you might need to extract subtitles file from mkv coz lingo-player can't fetch it

**Comment by ghost** [comment]
Yes, thank you, that's what I also thought to do. Unfortunately, it's lagging in 1080p anyway, so we have to wait for some fix...

**Comment by dezdem** [comment]
find 1080p with lower bitrate, it could help
